### PR TITLE
feat(fxa-71): Add outcomes for FxA-71 feature card

### LINKED ISSUE #131: Close out FxA-71 milestone
Including summarizing sucess/outcomes into the feature card for future reference.

### PR DESCRIPTION
**Pull request by rfk**
Fixes #131.  This moves the AMO migration card to "shipped" and adds a summary of the key "login completions" metric.  Are we interested in knowing any other measurements about the integration before declaring it "done"?

@shane-tomlinson r?; /cc @andymckay FYSA

### CODE REVIEW

File: features/shipped/FxA-71-AMO-migration-support/README.md [hunk 1]
```diff
@@ -31,6 +31,27 @@ who enter the FxA login flow from AMO.
...
+
+We shipped all outstanding work for this feature
+as of FxA train-58,
+which was shipped to production on 15 March 2016.
+As of the 23rd of March
+the completion rate for AMO logins is around 60%,
+while the combined completion rate for all oauth reliers
+is around 40%:
+
+![](completion-rate-screenshot.png)
```
Comment (vladikoff) [inline features/shipped/FxA-71-AMO-migration-support/README.md]: should have underscores here: 
`completion_rate_screenshot.png`

Comment (rfk) [inline features/shipped/FxA-71-AMO-migration-support/README.md]: whoops, thanks, fixed!

File: features/shipped/FxA-71-AMO-migration-support/README.md [hunk 2]
```diff
@@ -31,6 +31,27 @@ who enter the FxA login flow from AMO.
 
 These metrics are already available
 since we track them for other reliers.
+The specific query in datadog will be:
+
+* `login-attempts` = `fxa.content.screen.oauth.signin` + `fxa.content.screen.oauth.signup`
+* `successul-logins` = `fxa.content.screen.oauth.signin` + `fxa.content.screen.oauth.signup`
```
Comment (shane-tomlinson) [inline features/shipped/FxA-71-AMO-migration-support/README.md]: `successul`->`successful`.

It also looks like both formulas are identical. successful logins should have a `.success` suffix.

Comment (rfk) [inline features/shipped/FxA-71-AMO-migration-support/README.md]: thanks, fixed

**Comment by rfk** [comment]
I'm calling this done, let's close out the milestone

**Pull request merged**